CJSpohn/self-care-center
# Feature/user can favorite a message

Is this a fix or a feature?
Big feature

What is the change/fix?
The user now has the option to favorite a message by pressing the star icon next to the message. Once pressed, it saves the current message to its respective array (mantra or affirmation) and changes to gold. If the user presses "favorites" it takes them to a new page where their saved messages are displayed. A user can press "get more self care" to be returned to the main display. As new messages are generated, if they are already saved, the star image appears gold, if not, it appears black. If a user presses the star on an already saved message the star is returned to black, and that message is removed from their saved array.

Where should the reviewer start?
Oh boy. In main.js, to look at the added event listeners for the new buttons (star and favorites). And at their associated event handlers beginning at line 54.

How should this be tested?
When clicking the star button, you should be able to press "favorites" and see that same message appear under its respective heading. Upon returning to the main page, if you press the star again and go back to favorites, that message should no longer be there. After favoriting a message, when that message comes up randomly again it should already appear with a gold star indicating it has already been saved.

Screenshots(if appropriate)
![Screen Shot 2020-10-23 at 12 18 21 PM](https://user-images.githubusercontent.com/69563078/97039595-eaad4b00-1529-11eb-8632-febb75e49d7e.png)
![Screen Shot 2020-10-23 at 12 18 31 PM](https://user-images.githubusercontent.com/69563078/97039629-f4cf4980-1529-11eb-8f7a-bd6e6bc3eb3e.png)
